### DISCUSSION TITLE
Recover deleted schematics

### DISCUSSION THREAD

**Discussion by GR8MonkeyPirate** [maruohon/litematica#517] [Q&A] [answered]
Is there any way to recover deleted schematics? I made, pasted, and deleted a schematic from the in-game menu and was hoping there was a way to recover the file and put it back in my schematics folder. Any help is appreciated!

**Comment by GR8MonkeyPirate** [comment]
@maruohon I apologize for using a ping or if this is the wrong section but I'm impatient and desire an answer.

**Comment by maruohon** [comment, marked as answer]
Well pinging doesn't do anything extra, I just get an email in either case. And also it doesn't help if it in the middle of the night for me and I'm at sleep... ;)

As for the question, no, there is no way to get it back. Unless it's in the recycle bin on Windows? But I don't think the delete method I use would put it there, I think it just deletes it directly/permanently.

In general I don't even like the idea of a "recycle bin" or "trash bin" for files. Pretty much always when I delete files, I want them actually gone, and the space they took freed up for other things. So I only delete files when I know that I will not want or need it later.
